src/services/codeFixProvider.ts

```diff
@@ -127,6 +127,7 @@ function getDiagnostics({ program, sourceFile, cancellationToken }: CodeFixConte
     return [
         ...program.getSemanticDiagnostics(sourceFile, cancellationToken),
         ...program.getSyntacticDiagnostics(sourceFile, cancellationToken),
+        ...program.getDeclarationDiagnostics(sourceFile, cancellationToken),

```

Comment:
I'm curious how much this is going to slow things down; I'm surprised that this is the first instance of an emit based diagnostic getting a fix...

Reply:
We could only add it if isolatedDeclarations is enabled. But without this I think we won't get the code fix.